src/renate/updaters/learner.py

```diff
@@ -192,6 +207,16 @@ def training_step(
         """PyTorch Lightning function to return the training loss."""
         inputs, targets = self.training_step_unpack_batch(batch)
         outputs = self(inputs)
+        if self._mask_unused_classes:
+            if self._class_mask is None:
+                # Now is the time to repopulate the class_mask
+                self._class_mask = torch.tensor(
+                    complementary_indices(outputs.size(1), self._classes_in_current_task),
+                    device=outputs.device,
+                    dtype=torch.long,
+                )
+            # fill the logits with -inf
+            outputs.index_fill_(1, self._class_mask.to(outputs.device), -float("inf"))
```

Comment:
can we make all of the new code a function and only call this function here? then we can also reuse it for Offline-ER (new data and buffer data)